### ISSUE TITLE
`brew untap --force` returns `Error: invalid option: --force`

### ISSUE THREAD

**Issue by cheister** [Homebrew/brew#10511]
**`brew config` output**

HOMEBREW_VERSION: 2.7.7-140-gf33dc70
ORIGIN: https://github.com/Homebrew/brew
HEAD: f33dc70e856540ba300400e3ceb34ac84e49ee2b
Last commit: 2 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 85b3ffbe8a2f9cc346c89e174a069c3c288d246c
Core tap last commit: 49 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.fHWXJFSKia/org.macosforge.xquartz:0
HOMEBREW_EDITOR: atom --wait
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit haswell
Clang: 12.0 build 1200
Git: 2.30.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.1-x86_64
CLT: 12.4.0.0.1.1610135815
Xcode: N/A
XQuartz: 2.7.11 => /opt/X11

**`brew doctor` output**

Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed with `brew diy`.
You should find replacements for the following formulae:
  dirmngr
  delve
  camlp4
  godep
  protobuf@2.6
  gpg-agent
  phantomjs
  chromedriver

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  godep

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/cheister/.rvm/gems/ruby-2.6.6/bin/passenger-config

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libcrypto.1.0.0.dylib
  /usr/local/lib/libssl.1.0.0.dylib

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  p2
  gnupg@1.4
  bazel
  protobuf

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /usr/local/opt/p2

****

- [X] The `brew doctor` above contains no "Warning" lines.

**What were you trying to do (and why)?**

I'm trying to use `brew untap --force` but it is not listed as a valid option for untap.

**What happened (include all command output)?**

```
brew untap --force homebrew/core
Usage: brew untap tap [...]

Remove a tapped formula repository.

  -d, --debug                      Display any debugging information.
  -q, --quiet                      Make some output more quiet.
  -v, --verbose                    Make some output more verbose.
  -h, --help                       Show this message.
Error: invalid option: --force
```

**What did you expect to happen?**

Based on https://github.com/Homebrew/brew/issues/9419 I expected the tap to be untapped.

**Step-by-step reproduction instructions (by running `brew` commands)**

Run `brew untap --force`

**Comment by carlocab** [comment]
#9419 was closed by #9422, but this was reverted in #9483. FYI.

**Comment by dawidd6** [comment]
The functionality was added, but the flag was not. Will fix in PR above.
